free-programming-books-mm.md

```diff
@@ -10,6 +10,11 @@
 * [Saturngod's Books](http://books.saturngod.net)
 
 
+### Python
+
+* [Programming Basic](http://books.saturngod.net/programming_basic/index.html) - Saturngod
```

Comment:
The title of the resource on the page doesn't match the title given in the repository. Could they be synced up? ie "Programming For Beginner"

Also could you remove the `index.html` portion of the link as this is redundant.

Reply:
The author named like that.

Reply:
I mean the author of the book.

Reply:
Ahh I see, sorry. I visited the page again and see what I read wasn't the books name.
This is indeed my fault, thanks for correcting me.

Could you please still address the redundant index.html at the end of the URL?

Reply:
It also still could be nice to use the books full name, which is "Programming Basic with Python".

Reply:
anything still wrong?

Reply:
The URL looks good to me! Thank you!

As or the book name, I'm not sure if it's "Programming Basic with Python", or "Programming For Beginner", since the cover page doesn't correlate with the title at the top of the resource. I'd vouch this is good though.